framework/src/meshgenerators/BreakMeshByBlockGenerator.C

```diff
@@ -39,12 +40,20 @@ BreakMeshByBlockGenerator::validParams()
   params.addParam<bool>(
       "split_transition_interface", false, "Whether to split the transition interface by blocks.");
   params.addParam<bool>("add_interface_on_two_sides",
-                        false,
+                        true,
                         "Whether to add an additional interface boundary at the other side.");
   params.addParam<BoundaryName>(
       "interface_transition_name",
       "interface_transition",
       "the name of the interface transition boundary created when blocks are provided");
+
+  params.addRelationshipManager("ElementSideNeighborLayers",
+                                Moose::RelationshipManagerType::GEOMETRIC |
+                                    Moose::RelationshipManagerType::ALGEBRAIC |
+                                    Moose::RelationshipManagerType::COUPLING,
```

Comment:
```suggestion
                                Moose::RelationshipManagerType::GEOMETRIC,
```
This object only needs geometric ghosting; it doesn't partake in algebraic calculations